src/main/java/core/basesyntax/BalanceOperation.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax;
+
+import java.util.Map;
+
+public class BalanceOperation implements OperationHandler {
+    @Override
+    public void handle(FruitTransaction transaction, Map<String, Integer> storage) {
+        storage.put(transaction.getFruit(), transaction.getQuantity());
```

Comment:
When implementing the balance operation, make sure to check that the `transaction.getQuantity()` is not negative. If it is, you should throw a RuntimeException to prevent negative balances in the storage.